projects/igniteui-angular/src/lib/grids/summaries/grid-summary.ts

```diff
@@ -163,26 +162,25 @@ export class IgxNumberSummaryOperand extends IgxSummaryOperand {
     public operate(data: any[] = [], allData: any[] = [], fieldName?: string, locale: string = 'en-US',
         pipeArgs: IColumnPipeArgs = {}): IgxSummaryResult[] {
```

Comment:
We should remove the `locale` and `pipeArgs` parameters from the `operate` methods.